docs/user_guide/mitigation/preprocessing.rst

```diff
@@ -139,3 +139,100 @@ we would expect with :math:`\alpha=0.5`.
 .. figure:: ../../auto_examples/images/sphx_glr_plot_correlationremover_before_after_003.png
     :align: center
     :target: ../../auto_examples/plot_correlationremover_before_after.html
+
+
+.. _fair_representation_learner:
+
+Fair Representation Learner
+---------------------------
+
+:class:`~fairlearn.preprocessing.FairRepresentationLearner` is a preprocessing algorithm and
+a classifier that aims to learn a latent representation of the data that minimizes reconstruction
+error, while simultaneously obfuscating information about sensitive features. It was introduced in
+Zemel et al. (2013) :footcite:`pmlr-v28-zemel13`.
+
+The latent representation consists of :math:`K` prototype vectors
+:math:`\mathbf{v}_1, \dots, \mathbf{v}_K \in \mathbb{R}^d`, where :math:`d` is the dimension
+of the input data, and a stochastic transformation matrix :math:`\mathbf{M} \in \mathbb{R}^{n \times k}`
+that maps the input data :math:`\mathbf{X} \in \mathbb{R}^{n \times d}` to the latent representation.
+Each entry :math:`M_{nk}` is the softmax of the distances of the sample :math:`n` to the prototype
+vectors:
+
+.. math::
+
+    M_{nk} = \frac{\exp(-d(\mathbf{x}_n, \mathbf{v}_k))}{\sum_{k'=1}^{K} \exp(-d(\mathbf{x}_n, \mathbf{v}_{k'}))}
+
+The algorithm works by solving an optimization problem that balances the trade-off between
+classification accuracy, group and individual fairness, and reconstruction error. The objective
+function is thus defined as a weighted sum of three terms:
+
+.. math::
+
+    \lambda_x \times L_x + \lambda_y \times L_y + \lambda_z \times L_z
+
+where:
+
+:math:`L_x` is the reconstruction error term that writes as:
+
+.. math::
+
+    L_x = \frac{1}{N} \sum_{n=1}^{N} \| \mathbf{x}_n - \hat{\mathbf{x}}_n \|_2^2
+
+:math:`L_y` is the classification error term that is equal to the log loss:
+
+.. math::
+
+    L_y = -\frac{1}{N} \sum_{n=1}^{N} y_n \log(\hat{y}_n) + (1-y_n) \log(1-\hat{y}_n)
+
+:math:`L_z` is the demographic parity difference's approximation term that is defined as:
+
+.. math::
+
+    L_z = \frac{1}{K \times \binom{G}{2}} \sum_{k=1}^{K} \sum_{g < g'} \left| M_{gk} - M_{g'k} \right|
```

Comment:
There are other options here that would be more compatible with how we do things elsewhere. For example, we could use
\sum_{g=1}^G \sum_{k=1}^K | M_{gk} - M_{*k} |
where M_{*k} =\frac{1}{N} \sum_{n=1}^n M_{nk}. This would however not be compatible with the paper for G=2.